data/json/terrain.json

```diff
@@ -1881,6 +1890,7 @@
         "type" : "terrain",
         "id" : "t_chaingate_l",
         "name": "locked wire gate",
+        "description": "A gate for a chain link fence but this one has a padlock on the latch system.  With the right tools, cutting the metal fence is possible as well as picking the lock.  Or you could just examine the fence to see if it looks climbable.",
```

Comment:
`cutting the metal fence is possible as well as picking the lock.  Or you could just examine the fence to see if it looks climbable.`

Mix of passive and active voice. Consider e.g.

`With the right tools, you could cut the metal fence or pick the lock.  You could also examine the fence to see if it looks climbable.`